deployment.go

```diff
@@ -56,23 +56,11 @@ type DeploymentStepV1 struct {
 
 // DeploymentPlan is a collection of steps for application deployment
 type DeploymentPlan struct {
-	ID       string `json:"id"`
-	Version  string `json:"version"`
-	Original struct {
-		Apps         []*Application `json:"apps"`
-		Dependencies []string       `json:"dependencies"`
-		Groups       []*Group       `json:"groups"`
-		ID           string         `json:"id"`
-		Version      string         `json:"version"`
-	} `json:"original"`
-	Steps  []*DeploymentStep `json:"steps"`
-	Target struct {
-		Apps         []*Application `json:"apps"`
-		Dependencies []string       `json:"dependencies"`
-		Groups       []*Group       `json:"groups"`
-		ID           string         `json:"id"`
-		Version      string         `json:"version"`
-	} `json:"target"`
+	ID       string            `json:"id"`
+	Version  string            `json:"version"`
+	Original *Group            `json:"original"`
+	Target   *Group            `json:"target"`
+	Steps    []*DeploymentStep `json:"steps"`
```

Comment:
Shall we reconsider this refactoring : ) ? I encounter a scenario like the following to reference this "Group":
``` go
findAppInGroup(e.Plan.Original, targetAppId)

func findAppInGroup(group *gomarathon.Group, id string) {
	......
}
```

In current code, I will have to declare the function with the `group` being `interface{}`. It's not convenient for the following use.